src/scenes/app/reducers/actions.js

```diff
@@ -0,0 +1,98 @@
+import actionTypes from './actionTypes';
+import ToDoController from '../../../networking/controllers/ToDoController';
+import strings from '../../../localization/en/strings';
+import { deserializeItems, deserializeItem } from '../../../helpers/serializer';
+
+export const getTodo = () => {
+  return async (dispatch) => {
+    dispatch(showLoading());
+    try {
+      const response = await ToDoController.getToDo();
+      dispatch(successGetTodo(deserializeItems(response.data)));
+    } catch (error) {
+      dispatch(failureGetToDo(strings.generalError));
```

Comment:
Maybe it would be useful to be a little more descriptive with the error messages